assets/lib/share-link/share-link.js

```diff
@@ -1,25 +1,23 @@
 /**
  * By Elementor Team
  */
-( function( $ ) {
-	window.ShareLink = function( element, userSettings ) {
-		var $element,
-			settings = {};
+( function() {
+	window.ShareLink = function( eElement, userSettings ) {
+		let element = {};
```

Comment:
Initially it was undefined, and now it's an object. Is that correct?

Reply:
Why don't we use the eElement instead of element?
Can't we use the eElement object that is passed to this function?

Originally we needed to convert it from an `element` to `$element`, but now maybe we can use it as is?